src/popup/popup.html

```diff
@@ -104,7 +104,7 @@
         <h3 class="text-align-center vocab heading">Bookmarks</h3>
         <button class="delete-all-button vocab button negative-indicating dark-shaded normal-sized basic">
           <div class="content">
-            Delete All
+            Delete Selected Bookmarks
```

Comment:
Change the button from "Delete Selected Bookmarks" to "Delete". When the user presses the button without selecting any bookmarks, then a snack bar message is shown "No bookmarks selected".

Reply:
@makkoncept,  I had made changes you specified.